manifests/globals.pp

```diff
@@ -70,6 +71,13 @@
         /^5\./ => '8.1',
         default => undef,
       },
+      'RedHat' => '9.2',
+      default => $::operatingsystemrelease ? {
+        /^7\./ => '9.2',
+        /^6\./ => '9.2',
+        /^5\./ => '9.0',
+        default => undef,
+      },
```

Comment:
Ill get to work on this, ive just checked with our Servers, default for RHEL5 is 8.1, RHEL6 is 8.4 and RHEL7 is 9.2, I will change those values and try to format this similarily to how its handled for Fedora and see if that breaks anything